packages/db/testdata/config.yaml

```
@@ -1,4 +1,4 @@
-cluster_name: 'TestCluster'
-num_tokens: 1
-start_rpc: false
-endpoint_snitch: GossipingPropertyFileSnitch
+cluster_name: 'TestCluster'
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)